components/ModEditSubmissionTopbar.vue

```diff
@@ -22,11 +22,30 @@
             </button>
         </div>
         <div class="flex flex-row justify p-2 font-bold ">
+            <p
```

Comment:
This retry message appears 3 times if the request fails. To show it is retrying the mutation in 60s with a countdown timer